core-bundle/src/Resources/contao/classes/BackendTemplateTrait.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao;
+
+/**
+ * @author Leo Feyer <https://github.com/leofeyer>
+ *
+ * @internal
+ */
+trait BackendTemplateTrait
+{
+	/**
+	 * Return the locale string
+	 *
+	 * @return string
+	 */
+	protected function getLocaleString()
+	{
+		$container = System::getContainer();
+
+		return
+			'var Contao={'
+			. 'theme:"' . Backend::getTheme() . '",'
+			. 'lang:{'
+			. 'close:"' . $GLOBALS['TL_LANG']['MSC']['close'] . '",'
```

Comment:
Can you please restore the indentation here and in the method below?

Reply:
Oops, auto formatting. Fixed in 9c6dc45830523700c850d89bb07efe83dab9ca1f.